inference-engine/src/transformations/src/transformations/op_conversions/convert_gather_0d.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_gather_0d.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertGather0D, "ConvertGather0D", 0);
+
+ngraph::pass::ConvertGather0D::ConvertGather0D() {
```

Comment:
Please add functional tests for this transformation.

Reply:
Added